air/tests/test_module/issues/mod.rs

```diff
@@ -31,6 +31,7 @@ mod issue_300;
 mod issue_302;
 mod issue_304;
 mod issue_306;
+mod issue_310;
```

Comment:
let's file an issue and name it accordingly

Reply:
The `ap` join behavior issue is VM-316, but the VM-310 issue demonstrates the original problem that folex had.

Reply:
This list is based on github issues

Reply:
Ok, let me put it in the following way:
 - we have linear issues
 - we still have github issues
 
From the code perspective, list in 'air/tests/test_module/issues/mod.rs' is based on the github issues because they are public. And for this PR I propose to file a new issue in github and use its number here.

Reply:
Sorry, I see now.